compiler/il/OMRBlock.cpp

```diff
@@ -1708,6 +1708,19 @@ OMR::Block::getGlobalNormalizedFrequency(TR::CFG * cfg)
    return frequency;
    }
 
+/*
+ * An OSR induce block is used in postExecutionOSR to contain the OSR point's dead stores
+ * and to induce the transition. It will always have no successors and one exception
+ * successor to an OSR catch block.
+ */
+bool
+OMR::Block::isOSRInduceBlock(TR::Compilation *comp)
+   {
+   return comp->getOSRMode() == TR::voluntaryOSR
+      && self()->getExceptionSuccessors().size() == 1
```

Comment:
Do we really need the test that there must only be a single exception successor ? 
Is it possible that the induce block has a non OSR catch block as an exception successor in addition to an OSR catch block ?